kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -259,35 +253,13 @@ public static ResponsiveConfig responsiveConfig(final Map<?, ?> originals) {
    * mid-application, for example during state store init.
    */
   public static ResponsiveConfig loggedConfig(final Map<?, ?> originals) {
-
-    final Properties props = loadVersionPropertiesFromFile();
-    final String version = props.getProperty("git.build.version", "").trim();
-    final String commitId = props.getProperty("git.commit.id", "").trim();
-
-    if (!version.isEmpty() && !commitId.isEmpty()) {
-      LOG.info("Responsive Client version: {}", version);
```

Comment:
Basically moved the logging into ResponsiveKafkaStreams, and the actual version property file stuff into ClientVersionMetadata